CHANGELOG.md

```diff
@@ -30,6 +30,24 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### :rotating_light: Breaking changes
 - ...
 
+## [v0.19.0] - 2024-07-07
+
+### :rocket: Enhancements
+- [#1804](https://github.com/reviewdog/reviewdog/pull/1804) Support deleting
+  outdated comments in github-pr-review reporter. Note that it won't delete
+  comments if there is a reply considering there can be a meaningful
+  discussion.
+- [#1806](https://github.com/reviewdog/reviewdog/pull/1806) Add
+  -reporter=rdjson/rdjsonl which outputs rdjson/rdjsonl format to stdout. It
+  also changes the default behavior of -diff and -filter-mode for local
+  reporters. If -filter-mode is not provided (-filter-mode=default) and -diff
+  flag is not provided, reviewdog automatically set -filter-mode=nofilter.
+- [#1807](https://github.com/reviewdog/reviewdog/pull/1807) Add -reporter=sarif
+  which outputs SARIF format to stdout. You can upload the output SARIF to
+  GitHub and see code scanning alerts.
+- [#1809](https://github.com/reviewdog/reviewdog/pull/1809) Support posting
+  comments outside diff context as a file comment in github-pr-review reporter
+
```

Comment:
https://github.com/reviewdog/reviewdog/blob/041d96986a22b786bc26359307a31438a14394b7/CHANGELOG.md?plain=1#L7-L31

The unreleased section remains.
Therefore, I think it would be a good idea to replace it with the v0.19.0 section.

Reply:
Oops, thank you for the catch :+1: